pandas/tests/series/test_operators.py

```diff
@@ -660,6 +660,14 @@ def run_ops(ops, get_ser, test_ser):
         self.assertRaises(TypeError, lambda: td1 - dt1)
         self.assertRaises(TypeError, lambda: td2 - dt2)
 
+    def test_sub_single_tz(self):
+        # GH12290
```

Comment:
For completeness, you should also add the reverse `s2 - s1` and check that that returns the desired result of `Timedelta('-2days')`